src/main/java/seedu/address/model/grade/GradeEnum.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.grade;
+
+public enum GradeEnum {
+    A("A"), B("B"), C("C"), D("D"), E("E"),
+    F("F"), S("S"), U("U");
```

Comment:
You have used the Enum correctly! Just a small recommendation, sorry I kind of forget how secondary school grading works, but maybe the Enum can accepts values such as "A1", "A2" to make it as close to the real world context? Otherwise it's fine to retain as single alphabet! Not an alarming issue!